src/window.rs

```diff
@@ -214,6 +214,10 @@ impl Window {
             .or_else(|_| create_gl_window(window, &event_loop, true))?;
         window.show();
 
+        if window_config.start_maximized() {
```

Comment:
This check should come before `window.show()` to prevent flickering

Reply:
I tried this, `set_maximized` did not work before `window.show()`, not sure why

Reply:
This is because `set_maximized` is a request to the window manager to maximize a window. If you want to start a window as maximized you should use this instead:
https://docs.rs/glutin/0.14.0/glutin/struct.WindowBuilder.html#method.with_maximized

Reply:
if you mean something like this (this is what I started with originally)
https://github.com/doubleloop/alacritty/commit/0eeda942226506dd07a69091db227e9177dd9df2

It does not work..

Reply:
That's interesting, could be a bug in glutin or an issue with the window manager maybe. Since it's not really forced client-side the window manager always has the option to ignore this kind of stuff. Do you have the ability to test this on a different WM maybe?

If this is a glutin bug it should be fixed in glutin, if a workaround is necessery to make it work consistently that can be considered.